cdap-common/src/main/java/org/apache/twill/filesystem/FileContextLocation.java

```diff
@@ -80,6 +80,32 @@ public boolean createNew() throws IOException {
   }
 
   @Override
+  public boolean createNew(String permission) throws IOException {
```

Comment:
According to the comment in https://issues.cask.co/browse/CDAP-5328, we shouldn't need to copy the `FileContextLocation` class anymore.
Check if there's any changes in cdap that don't exist in twill. If nothing, we can probably just remove this class from CDAP.